### DISCUSSION TITLE
Add support fo LuaU

### DISCUSSION THREAD

**Discussion by osp54** [gudzpoz/luajava#178] [Ideas]
Сan you add support for [LuaU](https://github.com/luau-lang/luau)? It has compatibility with lua51(in embedding too), and is its typed superset.

**Comment by gudzpoz** [comment]
I created an issue (#182) for this. I don't think I currently have enough energy to work on this, since its incompatibilities and using CMake involve quite a bit of work. But you are always welcome to submit a PR for this and I am willing to provide any guidance :)
